invenio_users_resources/resources/users/config.py

```diff
@@ -35,3 +39,12 @@ class UsersResourceConfig(RecordResourceConfig):
     }
 
     request_search_args = SearchRequestArgsSchema
+
+    error_handlers = {
+        UserNotFoundError: create_error_handler(
+            lambda exc: HTTPJSONException(
+                code=404,
+                description="User not found.",
```

Comment:
```suggestion
                description=exc.args[0],
```
see my comment below :)